ulog/log.go

```diff
@@ -51,6 +53,9 @@ type Log interface {
 	// RawLogger returns underlying logger implementation (zap.Logger) to get around the ulog.Log interface
 	RawLogger() zap.Logger
 
+	// Sentry returns the underlying sentry hook
+	Sentry() *sentry.Hook
```

Comment:
People won't but i needed it for tests. Maybe you can suggest a better method?

Basically, there are tests in `service_test` that load a bunch of yaml and validate that sentry was created correctly with the new service.